test/extended/networking/util.go

```diff
@@ -158,7 +159,10 @@ func launchWebserverPod(f *e2e.Framework, podName string, nodeName string) (ip s
 	port := 8080
 	pod := &api.Pod{
 		ObjectMeta: api.ObjectMeta{
-			Name: podName,
+			Name:   podName,
```

Comment:
Consider applying this change upstream also so that kube's LaunchWebserverPod method can replace origin's version:

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/util.go#L3504